### PR TITLE
feature: Shortcut for collapse the left sidebar #692

### PR DESCRIPTION
**Pull request by naughtz**

### CODE REVIEW

**Comment by naughtz** [comment]
The CI/build failed because the `hotkey_manager` package requires keybinder-3.0 on linux.
I added `apt-get install keybinder-3.0` in install_linux.sh but it does not work.
I need help about how to install an apt package when CI/build.

**Comment by appflowy** [comment]
I saw you fix the keybinder issue. 👍🏻

File: frontend/app_flowy/lib/workspace/presentation/home/navigation.dart [hunk 1]
```diff
@@ -84,23 +88,32 @@ class FlowyNavigation extends StatelessWidget {
...
-                onPressed: () {
-                  notifier.value = false;
-                  ctx.read<HomeBloc>().add(const HomeEvent.collapseMenu());
-                },
-                iconPadding: const EdgeInsets.fromLTRB(2, 2, 2, 2),
-                icon: svgWidget("home/hide_menu", color: theme.iconColor),
-              ),
+              child: Tooltip(
+                  richMessage: TextSpan(children: [
+                    const TextSpan(text: "Open sidebar\n"),
```
Comment (appflowy) [inline frontend/app_flowy/lib/workspace/presentation/home/navigation.dart:102]: Should localize this string

File: frontend/app_flowy/lib/workspace/presentation/home/menu/menu.dart [hunk 2]
```diff
@@ -217,14 +217,22 @@ class MenuTopBar extends StatelessWidget {
...
-                width: 28,
-                onPressed: () => context
-                    .read<HomeBloc>()
-                    .add(const HomeEvent.collapseMenu()),
-                iconPadding: const EdgeInsets.fromLTRB(4, 4, 4, 4),
-                icon: svgWidget("home/hide_menu", color: theme.iconColor),
-              )
+              Tooltip(
+                  richMessage: TextSpan(children: [
+                    const TextSpan(text: "Close sidebar\n"),
```
Comment (appflowy) [inline frontend/app_flowy/lib/workspace/presentation/home/menu/menu.dart:222]: Should localize this string.  Check out this [link](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/translation)

**Pull request merged**